ansible/roles/kubernetes-addons/tasks/cluster-monitoring.yml

```diff
@@ -13,6 +13,7 @@
     url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/addons/cluster-monitoring/influxdb/{{ item }}
     dest="{{ local_temp_addon_dir }}/cluster-monitoring/{{ item }}.j2"
     force=yes
+    validate_certs=False
```

Comment:
pls remove and do an overall scrub on changes that are outside the scope of what's needed to add Contiv support.